### PR TITLE
maintenance(www): Simplify the way we structure and set up the /packages pages

### PR DESCRIPTION
**Pull request by tesseralis**
## Description

* Remove code related to packages from `utils/node/docs.js`.
* `official` field in `NPMPackage` to determine if it's an official gatsby plugin.
  * Instead of looking through local packages, we check that the owner of the package is `gatsbyjs`.
* Combine `template-docs-local-packages` and `templates-docs-remote-packages`.
* Get the markdown through `npmPackage.readme.childMarkdownRemark` instead of a separate query.
* Get rid of `MarkdownPageFooter` (the "Edit on GitHub" link and "Last Updated") since it doesn't make sense for plugin READMEs, and the information is already in the "View Plugin on GitHub" link.

## Motivation

* Decouple package code from the docs code to make it easier to factor dot-org into themes.
* Now that we're getting the READMEs of gatsby docs from NPM, we don't need to source the entirity of `packages` anymore — just the stuff we need for API files.

### CODE REVIEW

File: www/src/utils/node/docs.js [hunk 1]
```diff
@@ -193,15 +189,6 @@ exports.createPages = async ({ graphql, actions }) => {
             ...prevAndNext,
           },
         })
-      } else if (node.fields.package) {
```
Comment (tesseralis) [inline www/src/utils/node/docs.js:196]: This never actually gets called because the NPM packages don't go through the MDX transformer!

**Comment by tesseralis** [comment]
@AishaBlake this should be ready for a re-review now. [gatsby-plugin-manifest works](https://build-4a63945f-fbcb-4b84-b632-163c12104ff2.gtsb.io/packages/gatsby-plugin-manifest/) and clicking around it seems like both official and community plugins work.

**Comment by AishaBlake** [comment]
My internet cut out as I was reviewing this yesterday, so I sent a message in Slack on my phone. Adding my comments here now for completeness!

It looks like something is up with the way we're determining whether or not a plugin is official. As an example, `gatsby-source-filesystem` shows up as a community plugin even though it meets the criteria in the util function.

**Comment by tesseralis** [comment]
@AishaBlake is this testing locally or in the cloud build? The cloud build shows `source-filesystem` as official:

screenshot of gatsby-source-filesystem plugin page, showing 'Official Plugin'

**Pull request merged**